brianlong/www.validators.app
# 177618323-historical-SOL-prices-views

#### What's this PR do?
- Add views for sol prices

#### How should this be manually tested?
- Run `rails r script/sol_prices/coin_gecko_gather_historical_prices.rb` and `rails r script/sol_prices/ftx_gather_historical_prices.rb` unless you have the data
- Go to `/sol_prices` (you may need to run `yarn install` first)
- Check the charts

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/179624441

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
